nixos/doc/manual/development/replace-modules.xml

```diff
@@ -0,0 +1,63 @@
+<section xmlns="http://docbook.org/ns/docbook"
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+        xmlns:xi="http://www.w3.org/2001/XInclude"
+        version="5.0"
+        xml:id="sec-replace-modules">
+
+<title>Replace Modules</title>
+
+<para>Modules that are imported can also be disabled.  The option
+  declarations and config implementation of a disabled module will be
+  ignored, allowing another to take it's place.  This can be used to
+  import a set of modules from another channel while keeping the rest
+  of the system on a stable release.</para>
+<para><literal>disabledModules</literal> is a top level attribute like
+  <literal>options</literal> and <literal>config</literal>.  It
```

Comment:
There's not really an overview except for the [writing modules](http://nixos.org/nixos/manual/#sec-writing-modules) section. I could add it there, but I'm not sure if that's the best place.